botnim/query.py

```diff
@@ -187,3 +187,35 @@ def format_mapping(mapping: Dict, indent: int = 0) -> str:
     
     return "\n".join(result)
 
+def elastic_vector_search_handler(environment: str, bot_name: str, context_name: str, query: str, num_results: int = 3) -> str:
```

Comment:
This function would have been better off as a parameter for `run_query` (e.g. `output_format` = 'text' / 'data'). 
Another better option would have been to to add a `format_search_results` utility function.

Reply:
I hope I understood you correctly here, had to use my digital friend's help to clarify. so fixed, I hope.